### PR TITLE
Adjust Noko Import service

### PR DESCRIPTION
**Pull request by cleicar**
Hey guys

Fixes: https://github.com/ombulabs/pecas/issues/52

This is an attempt to fix Noko service to get entries from the past 2 weeks.

### CODE REVIEW

**Comment by cleicar** [comment]
Hey @etagwerker in order to use VCR we will have to create a fake Noko account so we don't share our private data in the VCR files. Would that be an issue? It looks like Noko doesn't have a free account, so let me know.

**Comment by lubc** [comment]
@cleicar This might be helpful for that: https://www.rubydoc.info/gems/vcr/VCR/Configuration:filter_sensitive_data

If that doesn't work I guess you can edit the VCR cassette and manually remove any sensitive information.

**Comment by etagwerker** [comment]
What @lubc said would be the best way to go. No need for a new account.

**Comment by cleicar** [comment]
@etagwerker @lubc can I have your review here? Thanks!

**Comment by cleicar** [comment]
@etagwerker can I have a new review? Thanks

File: spec/services/noko_client_spec.rb [hunk 1]
```diff
@@ -66,20 +57,22 @@
...
       end
+
+      context "when importing entries" do
+        let(:start_date) { Time.now.beginning_of_week.to_date - 1.week }
+
+        it "saves entries from the past two weeks" do
+          expect do
+            described_class.import_entries(start_date, end_date)
+          end.to change(Entry, :count).by(3)
+        end
```
Comment (etagwerker) [inline spec/services/noko_client_spec.rb:74]: @cleicar I'm having a hard time understanding how your change is actually solving the problem that we are seeing. 

How is this going to help with people submitting this week entries for the past week? 

I see that the rake task remains unchanged: https://github.com/ombulabs/pecas/blob/cdf659ad519f7f474fdb307f8e365bcda56eacf8/lib/tasks/importer.rake#L11-L21

Wouldn't we need to make the start_date further back in that rake task?

Comment (cleicar) [reply spec/services/noko_client_spec.rb:74]: Hey @etagwerker the purpose of my changes is to use a bigger interval of days when importing the entries. 

The importer task was already updated in my previous PR: https://github.com/ombulabs/pecas/pull/54/files#diff-3ad61c686e56f51add5f30435601d020R12

But after that we realized that the service wasn’t getting all the entries from all pages, so this is the goal for this PR.

Comment (etagwerker) [reply spec/services/noko_client_spec.rb:74]: @cleicar  makes sense now. Thanks!

**Pull request merged**